### PR TITLE
IU-20 | add search results per model

### PR DESCRIPTION
**Pull request by mikowhy**
ISSUE: https://github.com/unicef/iogt/issues/83
---

when surveys / polls / quizes...
- https://github.com/unicef/iogt/issues/5
- https://github.com/unicef/iogt/issues/6
- https://github.com/unicef/iogt/issues/38

... will be ready:
- new template (tag) for rendering should be added in `home/templates/home/tags/xxxxxxx.html`
- searchable models should be added https://github.com/mikowhy/iogt/blob/feat/IU-20-gruped-search-results/search/views.py#L9
- depend on tag implementation, it should be added to `search.html` template https://github.com/mikowhy/iogt/blob/feat/IU-20-gruped-search-results/search/templates/search/search.html#L24

### CODE REVIEW

File: iogt/settings/base.py [hunk 1]
```diff
@@ -9,8 +9,6 @@
 For the full list of settings and their values, see
 https://docs.djangoproject.com/en/3.1/ref/settings/
 """
-from django.utils.translation import gettext_lazy as _
```
Comment (mwrobel-merixstudio) [inline iogt/settings/base.py:12]: ok. I see - it was duplicated ;)

**Review by mwrobel-merixstudio — APPROVED** [review]

**Review by mwrobel-merixstudio — APPROVED** [review]

**Review by mwardak-mrx — CHANGES_REQUESTED** [review]
Some minor changes and it's good :)

**Comment by dannyparsons** [comment]
merged in #222

**Comment by mikowhy** [comment]
@ehmadzubair please let me know if unmerged commits (https://github.com/unicef/iogt/pull/122/files) should be committed as well

File: search/views.py [hunk 2]
```diff
@@ -1,35 +1,46 @@
...
-        search_results = paginator.page(1)
-    except EmptyPage:
-        search_results = paginator.page(paginator.num_pages)
-
-    return TemplateResponse(request, 'search/search.html', {
-        'search_query': search_query,
-        'search_results': search_results,
-    })
+    for search_group in search_groups:
+        search_results_no = 0
```
[the remark this answers is not in the document]
Comment (mikowhy) [reply search/views.py:20]: changed

File: search/views.py [hunk 3]
```diff
@@ -1,35 +1,46 @@
...
+from questionnaires.models import Survey, Quiz
 
 
 def search(request):
-    search_query = request.GET.get('query', None)
-    page = request.GET.get('page', 1)
+    search_groups = (Article, Section, Survey, Quiz)
+
+    search_query = request.GET.get("query")
+    page = request.GET.get("page", 1)
```
[the remark this answers is not in the document]
Comment (mikowhy) [reply search/views.py:14]: changed

**Comment by mikowhy** [comment]
unmerged commits in new PR -> https://github.com/unicef/iogt/pull/237

File: search/views.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (mwardak-mrx) [reply search/views.py:14]: Ah, right, I missed the fact that this is a function view and not a class-based DRF view. It's fine then 👍

**Pull request closed without merging**